api/content/data/front.rng

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<grammar ns="http://libero.pub" xmlns="http://relaxng.org/ns/structure/1.0"
+         xmlns:a="http://relaxng.org/ns/compatibility/annotations/1.0">
+
+    <include href="../../../core/data/core.rng"/>
+
+    <start>
+        <ref name="libero.front"/>
+    </start>
+
+    <div>
+
+        <a:documentation>front element</a:documentation>
+
+        <define name="libero.front">
+
+            <element name="front">
+                <ref name="libero.front.attributes"/>
+                <ref name="libero.front.content"/>
+            </element>
+
+        </define>
+
+        <define name="libero.front.attributes">
+            <ref name="libero.attributes.lang"/>
+        </define>
+
+        <define name="libero.front.content">
+
+            <interleave>
+                <ref name="libero.front.content.id"/>
+                <ref name="libero.front.content.title"/>
+            </interleave>
+
+        </define>
+
+    </div>
+
+    <div>
+
+        <a:documentation>front/id element</a:documentation>
+
+        <define name="libero.front.content.id">
+
+            <element name="id">
```

Comment:
Other things with identifiers will use an attribute, this is a little different (ie not going to end up as a HTML fragment) but maybe all parts (body etc) should have the ID so would need to be an attribute?

Reply:
What is the purpose of this _id_ element, is it a global identifier of this content or is it a local XML identifier (to refer to from other elements)? If it is the former, should it maybe have a more distinguished name?

Reply:
The same ID as used in the URI. `content-id`?

Reply:
Yes, `content-id` might be more clear.